recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "openmp" %}
```

Comment:
Got OpenCL on the brain? 😜 Side note: If you have any feedback on those OpenCL libraries, would definitely be interested to hear it.

Yeah, `llvm-openmp` sounds good. Guess `gcc` would be `libgomp` and Intel would be `libiomp`. So maybe there is no real conflict. Seems like Homebrew just called it `openmp` in the end. [1] Debian appears to go for `libomp`. [2] [3] Not sure about other distros.

1: https://github.com/Homebrew/homebrew-core/pull/3028
2: https://packages.debian.org/sid/libomp-dev
3: https://packages.ubuntu.com/search?keywords=libomp-dev